js/main.js

```diff
@@ -133,3 +152,16 @@ document.addEventListener('DOMContentLoaded', function (event) {
     viewSwapper(data.view);
   }
 });
+
+function clickEdit() {
+  if (event.target.tagName === 'A') {
+    viewSwapper(event.target.getAttribute('data-view'));
```

Comment:
Because of what I said above, this should become the else for the condition checking if the username property is empty. We always want to return if the element clicked isn't an anchor tag, but we only want to go to the user's requested view if they have user data 👍